jerry-core/api/jerry.c

```diff
@@ -1482,6 +1483,52 @@ jerry_create_string_sz (const jerry_char_t *str_p, /**< pointer to string */
   return ecma_make_string_value (ecma_str_p);
 } /* jerry_create_string_sz */
 
+/**
+ * Calculates the size of the given pattern and creates a RegExp object.
+ *
+ * @return value of the constructed RegExp object.
+ */
+jerry_value_t
+jerry_create_regexp (const jerry_char_t *pattern, /**< RegExp pattern */
+                     jerry_regexp_flags_t flags) /**< Optional RegExp flags */
```

Comment:
if i recall correctly, param docstrings start with lowercase letters, usually. (it is fine to capitalize RegExp though.)